services/github/src/handlers.js

```diff
@@ -492,7 +492,7 @@ async function statusHandler(message) {
   try {
     const taskDefinition = await this.queueClient.task(taskId);
     const {customCheckRun} = (taskDefinition.extra && taskDefinition.extra.github) || {};
-    const {textArtifactName} = customCheckRun || CUSTOM_CHECKRUN_TEXT_ARTIFACT_NAME;
```

Comment:
This misses the case where `customCheckRun` is undefined, such as when there's no `task.extra`.

Reply:
Isn't that taken care of on the previous line? `const {customCheckRun} = (taskDefinition.extra && taskDefinition.extra.github) || {};`

Reply:
Checked - it doesn't. Now it should be better